curvefs/src/metaserver/storage/rocksdb_storage.h

```diff
@@ -92,6 +95,13 @@ class RocksDBStorage : public KVStorage, public StorageTransaction {
                 const std::string& key,
                 const ValueType& value) override;
 
+    Status HSetDeleting(const std::string& name,
```

Comment:
The interfaces added is unnecessary, rocksdb_storage only need the most basic interface, and you should deal the logic upper layer